### PR TITLE
UI/UX + landing

### PR DESCRIPTION
**Pull request by Rahmteen**
Improved UI and UX and included a landing page. The routing for the entire application is processed in App.tsx. I also resolved the Monaco file retrieval bug that was appearing in the electron console. 

Best, 
Ramtin

### CODE REVIEW

**Comment by kspicerr** [comment]
Good stuff Ramtin!  Love the Monaco fix!

File: src/pages/Home/Home.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        <div 
+        key="c" 
+        data-grid={{
+          x: 2, 
+          y: 0, 
+          w: 1, 
+          h: 18.5,
+          minW: 1, 
+          maxW: 1,
+          static: false,
```
Comment (Rahmteen) [inline src/pages/Home/Home.tsx]: revised static for monaco to true. This will allow edits without movement animation. Thanks @jcspeare

**Comment by jcspeare** [comment]
Digging the landing page and home pages, and routing. Super snappy! 
Thank you for the hoverable dropdown navbar <3

**Review by Rahmteen — COMMENTED** [review]
Thanks Kim!

**Pull request merged**